projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -153,6 +153,7 @@ private static long fromDateTime(DateTime value) {
         Integer resourcesMemoryLimit;
         OffsetDateTime deployedDate;
         String deployedBy;
+        String containerTerminationReason;
```

Comment:
nit: maybe change `terminationReason` to `jobTerminationReason` here as well.

Reply:
I was thinking of doing it, but the attribute is directly invoked in a large number of places, and the cost of the change seemed unreasonable.